src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
```

Comment:
Same as before. There is no need to import the whole react library into the file from the newest version of it.